tools/jdbc/src/main/java/org/duckdb/DuckDBPreparedStatement.java

```diff
@@ -155,7 +155,7 @@ public ResultSet executeQuery() throws SQLException {
 
 	@Override
 	public int executeUpdate() throws SQLException {
-		if (!is_update) {
+		if (!returnsChangedRows) {
```

Comment:
This means we can't use `executeUpdate` with catalog statements (e.g. `CREATE TABLE` - they return nothing) or with `INSERT INTO RETURNING` statements (which return a result set). Is that what we want?

Reply:
You're correct yes, turns out I missed a line in the specification - `executeUpdate` should accept a query that updates rows, or returns nothing

> Executes the given SQL statement, which may be an INSERT, UPDATE, or DELETE statement or an SQL statement that returns nothing, such as an SQL DDL statement.

See https://docs.oracle.com/javase/7/docs/api/java/sql/Statement.html#executeUpdate(java.lang.String)

I've fixed this, and reverted the test changes